geometry_msgs/Vector3 q_des
geometry_msgs/Vector3 dq_des
geometry_msgs/Vector3 ddq_des
